cursors.yaml

```diff
@@ -101,7 +101,7 @@ Cursors:
 			Length: 9
 		deploy-blocked:
 			Start: 119
-		undeploy:
+		undeploy-unused:
```

Comment:
Readded it as requested, just as `undeploy-silver` now, as it's color is silver. I tend to agree to @Mailaender that this a) won't hurt and b) the artwork is still there and could be used.